Misc/NEWS.d/next/Library/2025-01-20-16-02-38.gh-issue-129064.JXasgJ.rst

```diff
@@ -0,0 +1,2 @@
+Deprecate :func:`sysconfig.expand_makefile_vars`, in favor of using
```

Comment:
Should we also add it to `pending-removal-in-3.16.rst`?

Reply:
Yes, good catch, thanks!